addons/im_livechat/static/src/core/web/composer_patch.js

```diff
@@ -4,6 +4,28 @@ import { _t } from "@web/core/l10n/translation";
 import { patch } from "@web/core/utils/patch";
 
 patch(Composer.prototype, {
+    get navigableListProps() {
+        const props = super.navigableListProps;
+        if (!this.hasSuggestions) {
```

Comment:
This condition could be removed in favor of optional chaining in the switch case below (`this.suggestion?`). Ultimately, we always return `props`; we simply want to enrich it if the type is `Chatbot`. I think this change would slightly reduce the complexity of this override.